core/src/test/java/tech/tablesaw/joining/DataFrameJoinerTest.java

```diff
@@ -905,6 +905,7 @@ public void innerJoinStudentInstructorOnAge() {
     assert (joined
         .columnNames()
         .containsAll(Arrays.asList("T2.ID", "T2.City", "T2.State", "T2.USID", "T2.GradYear")));
+    System.out.println(joined.printAll());
```

Comment:
shouldn't have `println` in checked in code

Reply:
OK. I see.: ) I forget to delete this.